app/script_variant.py

```diff
@@ -43,31 +49,39 @@ def romanize_japanese_name_by_name_dict(word):
     This method romanizes japanese name by using name dictionary.
     If word isn't found in dictionary, this method doesn't
     apply romanize.
+    This method can return multiple romanizations.
```

Comment:
Non-Japanese people may not know that there are multiple ways to read the same kanji name. Please explain it here.